demos/projects/ESPRESSIF/adu/README.md

```diff
@@ -245,30 +250,14 @@ The resulting binary `azure_iot_freertos_esp32.bin` should be located in the bui
 
 ### Generate the ADU Update Manifest
 
-Navigate (on the new ESP-IDF console) to `C:\ADU-update` directory:
+Open PowerShell.
 
-Clone the ADU toolset.
-
-```bash
-git clone https://github.com/Azure/iot-hub-device-update
-```
+Navigate to `C:\ADU-update` directory.
 
-Generate the update manifest using **powershell**.
+Run the following command:
 
 ```powershell
-Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process
-
-Import-Module .\iot-hub-device-update\tools\AduCmdlets\AduUpdate.psm1
-
-$updateId = New-AduUpdateId -Provider "Contoso" -Name "ESP32-Azure-IoT-Kit" -Version 1.1
-
-$compat = New-AduUpdateCompatibility -Properties @{ deviceManufacturer = 'ESPRESSIF'; deviceModel = 'ESP32-Azure-IoT-Kit' }
-
-$installStep = New-AduInstallationStep -Handler 'microsoft/swupdate:1'-HandlerProperties @{ installedCriteria = '1.1' } -Files C:\ADU-update\azure_iot_freertos_esp32-v1.1.bin
-
-$update = New-AduImportManifest -UpdateId $updateId -Compatibility $compat -InstallationSteps $installStep
-
-$update | Out-File "./$($updateId.provider).$($updateId.name).$($updateId.version).importmanifest.json" -Encoding utf8
+az iot du update init v5 --update-provider Contoso --update-name ESP32-Azure-IoT-Kit --update-version 1.1 --compat deviceModel=ESP32-Azure-IoT-Kit deviceManufacturer=ESPRESSIF --step handler=microsoft/swupdate:1 properties='{\"installedCriteria\":\"1.1\"}' --file path=./azure_iot_freertos_esp32-v1.1.bin > ./Contoso.ESP32-Azure-IoT-Kit.1.1.importmanifest.json
```

Comment:
I was hoping the update to using the cli would remove the need to update the version number in a bunch of places :( I know before we had talked about using a script or something to simplify switching version numbers - should we consider doing something like that again?

Reply:
Could be something like

```powershell
$version = 1.1
az iot du update init v5 --update-provider Contoso --update-name ESP32-Azure-IoT-Kit --update-version $version --compat deviceModel=ESP32-Azure-IoT-Kit deviceManufacturer=ESPRESSIF --step handler=microsoft/swupdate:1 properties='{\"installedCriteria\":\"$version\"}' --file path=./azure_iot_freertos_esp32-v1.1.bin > ./Contoso.ESP32-Azure-IoT-Kit.1.1.importmanifest.json
```

I'm forgetting if that's powershell syntax or not but that's the basic idea.

Reply:
I was intending on keeping it a single line, to make it simple for customers to copy and run it. 
Since this is a sample, I rather keep the syntax explicit in showing the version, so people can clearly see what information goes where, instead of having to check where the variable $version (as suggested by Dane) is mentioned.
As I understand, your comment is guided by a desire to make this guidance more "productized" (with optimizations like what you proposed), but I believe we better keep this documentation written a more with an "educational" aspect.
If the team really prefers we do as suggested, I will for sure reconsider.